x-pack/plugins/ingest_manager/server/services/output.ts

```diff
@@ -13,7 +13,7 @@ const SAVED_OBJECT_TYPE = OUTPUT_SAVED_OBJECT_TYPE;
 
 class OutputService {
   public async ensureDefaultOutput(soClient: SavedObjectsClientContract) {
-    const outputs = await soClient.find<OutputSOAttributes>({
+    const outputs = await soClient.find<Output>({
```

Comment:
I would have expected this remain

```typescript
const outputs = await soClient.find<OutputSOAttributes>({
```

but I can see it fixes the type errors. @nchaulet you're more familiar with this than me. WDYT?

Reply:
@jfsiii I think you are commenting an outdated commit, I reverted that change

Reply:
@nchaulet it was from 14 hours ago https://github.com/elastic/kibana/commit/b49dad5c9ff645a1c99ba96cfcd4befe186d8dc3

Reply:
@nchaulet I committed this change to fix type error. feel free to tweak typings